WordPress/src/test/java/org/wordpress/android/ui/prefs/timezone/SiteSettingsTimezoneViewModelTest.kt

```diff
@@ -0,0 +1,56 @@
+package org.wordpress.android.ui.prefs.timezone
+
+import android.content.Context
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mock
+import org.wordpress.android.BaseUnitTest
+import org.wordpress.android.viewmodel.ResourceProvider
+
+class SiteSettingsTimezoneViewModelTest : BaseUnitTest() {
+    @Mock lateinit var resourceProvider: ResourceProvider
+    @Mock lateinit var context: Context
+
+    private lateinit var viewModel: SiteSettingsTimezoneViewModel
+
+    @Before
+    fun setUp() {
+        viewModel = SiteSettingsTimezoneViewModel(resourceProvider)
+    }
+
+    @Test
+    fun testSearchTimezones() {
+        viewModel.timezones.observeForever {}
```

Comment:
Just for the sanity check and to test the behavior of observer exactly, would you mind assigning result to an object, and asserting it, instead of checking `.value` ?

Reply:
Updated to assign result to an object and assert it